java/src/main/java/org/rocksdb/RocksDB.java

```diff
@@ -688,7 +711,9 @@ public ColumnFamilyHandle createColumnFamily(
     final List<ColumnFamilyHandle> columnFamilyHandles =
         new ArrayList<>(cfHandles.length);
     for (int i = 0; i < cfHandles.length; i++) {
-      columnFamilyHandles.add(new ColumnFamilyHandle(this, cfHandles[i]));
+      final ColumnFamilyHandle columnFamilyHandle = new ColumnFamilyHandle(this, cfHandles[i]);
+      ownedColumnFamilyHandles.add(columnFamilyHandle);
```

Comment:
Please move `ownedColumnFamilyHandles.add(columnFamilyHandle);` after the loop and change to:

`ownedColumnFamilyHandles.addAll(columnFamilyHandles);`

Reply:
Likewise for the second `createColumnFamilies` method below